### DISCUSSION TITLE
cs/system/cs1/topic1/

### DISCUSSION THREAD

**Discussion by giscus** [TonyCrane/note#5] [General]
# cs/system/cs1/topic1/

鹤翔万里（TonyCrane）的笔记本

https://note.tonycrane.cc/cs/system/cs1/topic1/

**Comment by sjiang-uuu** [comment]
本文3.5节末尾

结合律天然蕴含于群结构中。

群是一组资料：$(G, +)$，其中 $G$ 非空，而映射 $+：G\times G\rightarrow G$满足结合律、幺元与逆元的存在性。

对于定义的二元运算若满足交换性质，称该群为Abelian Group，即交换群。

因此末尾对浮点数类型为阿贝尔群的断言是错误的。

爱来自想去ZJU图灵26的小菜鸡（能有学上吗求求了
